CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.11.0 / 2017-07-20
+
+Warning: this release deprecates the previously used storage definition in favor of upstream PersistentVolumeClaim templates. While this should not have an immediate effect on a running cluster, Prometheus object definitions that have storage configured need to be adapted. The previously existing fields are still there, but have no effect anymore.
```

Comment:
Can we point people to an example how this storage definition changes are done best?